js/packages/mobile-wallet-adapter-walletlib/src/_DEPRECATED_requests.ts

```diff
@@ -36,6 +36,13 @@ export enum MobileWalletAdapterServiceRequestEventType {
 // Requests that come from the dApp for Authorization, Signing, Sending services.
 export abstract class MobileWalletAdapterServiceRequest {
```

Comment:
We can probably delete all these deprecated files, especially if the full example app UI is working now.